src/modules/networking/tests/NetworkingTests.cpp

```diff
@@ -486,6 +497,40 @@ namespace OSConfig::Platform::Tests
             "DNSSEC supported: no\n"
             "DNS Domain: mshome.net\n";
 
+        std::string testCommandOutputMultipleDnsServersPerLine =
+            "Global\n"
+            "LLMNR setting: no\n"
+            "MulticastDNS setting: no\n"
+            "DNSOverTLS setting: no\n"
+            "DNSSEC setting: no\n"
+            "DNSSEC supported: no\n"
+            "Current DNS Server: 1.1.1.1\n"
+            "DNS Servers: 1.1.1.1 8.8.8.8\n"
+            "DNSSEC NTA: 10.in-addr.arpa\n"
+            "Link 1 (docker0)\n"
+            "Current Scopes: DNS\n"
+            "DefaultRoute setting: yes\n"
+            "LLMNR setting: yes\n"
+            "MulticastDNS setting: no\n"
+            "DNSOverTLS setting: no\n"
+            "DNSSEC setting: no\n"
+            "DNSSEC supported: no\n"
+            "Current DNS Server: 172.29.64.1\n"
+            "DNS Servers: 172.29.64.1 10.20.20.20\n"
+            "DNS Domain: mshome.net\n"
+            "Link 2 (eth0)\n"
+            "Current Scopes: DNS\n"
+            "DefaultRoute setting: yes\n"
+            "LLMNR setting: yes\n"
+            "MulticastDNS setting: no\n"
+            "DNSOverTLS setting: no\n"
+            "DNSSEC setting: no\n"
+            "DNSSEC supported: no\n"
+            "Current DNS Server: 172.127.41.161\n"
+            "DNS Servers: 172.127.41.161 10.20.20.20\n"
```

Comment:
Unit tests should not use real bits, including network addresses. Possible to mock? Otherwise unit tests become functional.